### DISCUSSION TITLE
我的问题已解决，谢谢服务团队。

### DISCUSSION THREAD

**Discussion by cryospherewang** [getlantern/lantern-forum-cn#228] [Q&A] [answered]
希望稳定性将来可以改善，到时候再见。

请其他用户移步自己的新帖子，不然我会收到不必要的通知，谢谢。正在考虑删帖。

祝好
Bing

**Comment by MaxMKMax** [comment]
@cryospherewang 您好。对于造成的不便，我们深表歉意。我会帮你直接联系负责人。我认为不是因为支持人员忽略了您的电子邮件，我们这边很有可能因为被阻止接收您的电子邮件而没有收到您的电子邮件。

我会帮您直接向负责人处理您的案件，请给我们几天时间。抱歉让您久等了，非常感谢。

**Reply by MaxMKMax** [reply]
@cryospherewang 您好. 请问您收到回复了吗?

**Reply by cryospherewang** [reply]
您好，谢谢回复，不知道您有没有收到邮件回复。注册邮箱是cryospherewang@163.com，谢谢

**Reply by MaxMKMax** [reply]
@cryospherewang 您好. 不客气! 我们应该是没收到. 没事, 谢谢您的邮箱. 我直接帮您交给支付团队吧. 抱歉让您久等了!

**Reply by cryospherewang** [reply]
谢谢

**Comment by MaxMKMax** [comment]
@cryospherewang 您可以向我提供您购买并注册付费版本的电子邮件地址吗？我需要将其分享给付款团队以进行退款。

**Comment by cryospherewang** [comment]
***@***.*** 之前回复过您，可能又被block 了。Many thanksBing Sent from my iPhoneOn 13 Mar 2024, at 18:37, Mike K ***@***.***> wrote:﻿
@cryospherewang 您好. 请问您收到回复了吗?

**Reply by MaxMKMax** [reply]
@cryospherewang 您好. 不客气! 我们应该是没收到. 没事, 谢谢您的邮箱. 我直接帮您交给支付团队吧. 抱歉让您久等了!

**Reply by MaxMKMax** [reply]
@cryospherewang 你好。您的退款流程面临一些问题。我们的负责人正在解决您的问题。请5天后查看您的电子邮件。谢谢。

**Reply by cryospherewang** [reply]
您好，请问是什么问题呢？

**Comment by hjwremix** [comment]
你好，我也无法使用，发送了退款申请给邮箱，一直无反映。需要回复

> @cryospherewang 您好. 不客气! 我们应该是没收到. 没事, 谢谢您的邮箱. 我直接帮您交给支付团队吧. 抱歉让您久等了!

**Reply by MaxMKMax** [reply]
@hjwremix 您好，很抱歉给您带来不便。您发退款申请的邮件我们这里的部门是看不到的. 但没关系, 我会帮您再度提醒我们的支付部门.

请问您同时也可以提供我购买单据吗? 我需要通知付款团队，谢谢。

**Comment by hjwremix** [comment]
你好，这是我的付款收据。烦请协助通知付款团队，操作退款，谢谢。


我使用的专业版用于windows系统，使用不便，经常无法连接。










在 2024-03-17 23:46:02，"Mike K" ***@***.***> 写道：

@hjwremix 您好，很抱歉给您带来不便。我可以知道有什么问题吗？你能让我知道吗？您可以描述付费版本无法使用的具体问题。

还可以让我知道您正在使用什么设备吗？比如IOS、Android还是window等？

请同时向我提供您的退款收据。我需要通知付款团队，谢谢。

**Reply by MaxMKMax** [reply]
@hjwremix 你好。我不知道为什么我看不到你的收据。图像显示为空, 你能再重发一次吗？另请告诉我您注册专业版的电子邮件地址。谢谢。

**Comment by MaxMKMax** [comment]
@cryospherewang 您好! 好的，我会帮您转达给我们的负责人。抱歉，周末团队处于离线状态。他们将于今晚按照美国时区上线。我现在就通知他们，并帮助您今晚再次跟随他们。会尽快帮您解决。抱歉让您久等了。谢谢!

**Comment by hjwremix** [comment]
我的注册邮箱为： ***@***.***


购买收据也一并于附件，请查看


















在 2024-03-18 23:44:34，"Mike K" ***@***.***> 写道：

@hjwremix 你好。我不知道为什么我看不到你的收据。图像显示为空, 你能再重发一次吗？另请告诉我您注册专业版的电子邮件地址。谢谢。

**Comment by MaxMKMax** [comment]
<img width="479" alt="image" src="https://github.com/getlantern/lantern-forum-cn/assets/155475142/e9eb857b-b5bc-4dbb-8e83-597237117a64">

@hjwremix 您好. 我看到是空的. 而且邮件 ****@***** 这样的. 你能直接截图给我您的邮件吗? 像这样. 

<img width="108" alt="image" src="https://github.com/getlantern/lantern-forum-cn/assets/155475142/407e3ac2-0443-4e35-bdab-450aeeced870">

**Reply by hjwremix** [reply]
退款进度怎么样了？？

**Comment by hjwremix** [comment]
这是我之前发送的邮件截图，请查收！


























At 2024-03-19 20:32:16, "Mike K" ***@***.***> wrote:

image.png (view on web)

@hjwremix 您好. 我看到时空的. 而且有件事 @* 这样的. 你能直接截图给我您的邮件吗? 像这样.

image.png (view on web)

**Reply by hjwremix** [reply]
![image (1)](https://github.com/getlantern/lantern-forum-cn/assets/163745123/15297ec4-750e-4754-81fc-d211dc8aff9e)

![image](https://github.com/getlantern/lantern-forum-cn/assets/163745123/a5284a33-ad75-4915-9a99-6c74cbf24af7)

支付截图请查看。我的专业版电子邮件地址为：hjwremix@163.com，我发邮件给support@getlantern.org，始终无人回复，请问我的退款进展到哪了？

**Comment by hjwremix** [comment]
我已反复告知，购买了专业版后，一直无法上网，需要申请退款！！！


















在 2024-03-21 14:35:45，"cryospherewang" ***@***.***> 写道：

您好，请问是什么问题呢？

**Reply by MaxMKMax** [reply]
@cryospherewang 

您好, 请问您可以向我们的支付团队发送电子邮件吗？因为我们想保护您的个人信息和隐私。因此，最好直接向我们的支付团队发送电子邮件申请退款。

电子邮件：support@getlantern.org

**Reply by cryospherewang** [reply]
谢谢回复，邮件无法发送，只能通过 helpdesk 界面发送消息。目前我已经打不开 helpdesk 界面了，所以不知道处理进度如何了。

**Reply by MaxMKMax** [reply]
@cryospherewang 亲, 不客气呢. 如果您真的处理不了再来找我. 我从这里帮您弄. 
由于我不是支付部门的. 所以我们一般会让你们直接发邮件, 尤其是退款的事. 但请您放心, 如果真有需要. 我这里还是可以帮您直接对接另一个部门的. 谢谢您支持蓝灯. 
希望以后有机会您再回来!

**Reply by cryospherewang** [reply]
谢谢，我还是没收到回复诶

**Reply by MaxMKMax** [reply]
@cryospherewang 您好. 我已经帮您问第三次了. 看来客服那里最近超级忙. 我已经帮您追了刚才. 
请您等等啊. 不好意思!

**Reply by MaxMKMax** [reply, marked as answer]
@cryospherewang 支付团队已经帮您弄好退款了. 系统需要几天才能完成. 您应该会收到邮件.

**Reply by cryospherewang** [reply]
谢谢！已经收到退款了。

**Comment by xybModel** [comment]
从2月续订至现在，专业版一直无法使用，软件界面报告了好几次问题，都无人搭理，麻烦请退款。

**Reply by MaxMKMax** [reply]
@xybModel 您好, 如果您是受到授权码问题影响的用户，我们将通过重新延长您的专业版使用时间来补偿您。现在iOS系统暂时无法登陆. 我们正在弄新版本了, 很快可以更新了. 请您耐心等待. 非常抱歉影响了您的体验感!

**Reply by xybModel** [reply]
我从续费至现在，一直尝试用win版使用，软件显示“已连接：服务器位置SG”，位置有时跳转到US、HK等，但谷歌等无法正常使用，好多次没人回复。我也曾尝试不定期更新软件，包括删除默认文件夹中的信息，并重新登陆，但一直无法科学上网。麻烦请退款吧。

**Reply by MaxMKMax** [reply]
@xybModel 您好, 请问您可以向我们的支付团队发送电子邮件吗？因为我们想保护您的个人信息和隐私。因此，最好直接向我们的支付团队发送电子邮件申请退款。请您提供一下购买单据!

电子邮件：support@getlantern.org

**Comment by kulamata01** [comment]
这是我的专业版购买邮箱地址guanqun2004@msn.com,申请退款,请帮忙尽快处理谢谢

**Reply by MaxMKMax** [reply]
@kulamata01 您好, 请问您可以向我们的支付团队发送电子邮件吗？因为我们想保护您的个人信息和隐私。因此，最好直接向我们的支付团队发送电子邮件申请退款。请您提供一下购买单据!

电子邮件：support@getlantern.org

**Comment by kulamata01** [comment]
***@***.***
支付凭证在附件的截图
________________________________
发件人: Mike K ***@***.***>
发送时间: 2024年3月25日 13:51
收件人: getlantern/lantern-forum-cn ***@***.***>
抄送: kulamata01 ***@***.***>; Mention ***@***.***>
主题: Re: [getlantern/lantern-forum-cn] 连续一周了发退款软件都没人理 (Discussion #228)


@kulamata01<https://github.com/kulamata01> 您好, 请问您可以向我们的支付团队发送电子邮件吗？因为我们想保护您的个人信息和隐私。因此，最好直接向我们的支付团队发送电子邮件申请退款。请您提供一下购买单据!

***@***.******@***.***>

―

**Reply by MaxMKMax** [reply]
@kulamata01 您好. 请您发图和邮件给我们的客服. 谢谢您. 
support@getlantern.org

**Comment by kulamata01** [comment]
没有动静
________________________________
发件人: hjwremix ***@***.***>
发送时间: 2024年3月27日 5:31
收件人: getlantern/lantern-forum-cn ***@***.***>
抄送: kulamata01 ***@***.***>; Mention ***@***.***>
主题: Re: [getlantern/lantern-forum-cn] 连续一周了发退款软件都没人理 (Discussion #228)


退款进度怎么样了？？

―

**Comment by kulamata01** [comment]
已经在周一就发了
________________________________
发件人: Mike K ***@***.***>
发送时间: 2024年3月26日 17:10
收件人: getlantern/lantern-forum-cn ***@***.***>
抄送: kulamata01 ***@***.***>; Mention ***@***.***>
主题: Re: [getlantern/lantern-forum-cn] 连续一周了发退款软件都没人理 (Discussion #228)


@kulamata01<https://github.com/kulamata01> 您好. 请您发图和邮件给我们的客服. 谢谢您.
***@***.******@***.***>

―

**Reply by MaxMKMax** [reply]
@kulamata01 嗯嗯好的. 我帮您问看.
